apps/explorer/src/components/validator/ValidatorStats.tsx

```diff
@@ -161,6 +161,15 @@ export function ValidatorStats({
                                     tooltip="Latest Narwhal round for this epoch."
                                     unavailable
                                 />
+                                <Stats
+                                    label="Next Epoch Gas Price"
+                                    tooltip="Estimated gas fees in the upcoming epoch."
```

Comment:
This sounds wrong, this should be their vote for the gas price in the next epoch, right?

Reply:
updated the wording a bit - but I pulled this from here: https://github.com/MystenLabs/sui/blob/f389b8e83e7e408039992a577dfb723d13ba1675/crates/sui-framework/sources/governance/validator.move#L134